content/sql/concepts/window-functions/terms/lead/lead.md

```diff
@@ -0,0 +1,85 @@
+---
+Title: 'LEAD()'
+Description: 'Allows access to rows after the target value at an offset.'
+Subjects:
+  - 'Data Science'
+  - 'Computer Science'
+Tags:
+  - 'Functions'
+  - 'PostgreSQL'
+  - 'MySQL'
+  - 'SQLite'
+CatalogContent:
+  - 'paths/analyze-data-with-sql'
+---
+
+The **`LEAD()`** window function takes a column and a positive integer offset as arguments and returns the value of the cell in that column that is the specified number of rows after the current row. A third argument can be added to fill cells that do not have a corresponding row.
+
+## Syntax
+
+```pseudo
+SELECT LEAD(column1_name, offset) OVER(
+  ORDER BY column2_name
+)
+FROM table;
+```
+
+The `ORDER BY` clause is required. There is also a third, optional argument that sets the value of cells where the corresponding row does not exist.
+
+```pseudo
+SELECT LEAD(column1_name, offset, default) OVER(
+  ORDER BY column2_name
+)
+FROM table;
+```
+
+The third argument will be `NULL` by default.
```

Comment:
Can we make this a bullet point along with what's written on line 27 (let's create a few points to explain the different parameters here)

Reply:
Applied changes.